app/RouteHandlers/ApplyForMembershipHandler.php

```diff
@@ -33,34 +34,38 @@ public function __construct( FunFunFactory $ffFactory, Application $app ) {
 		$this->app = $app;
 	}
 
-	public function handle( Request $request ): Response {
-		if ( !$this->isSubmissionAllowed( $request ) ) {
+	public function handle( Request $httpRequest ): Response {
```

Comment:
We could take to opportunity of having to touch this code to convert this "Handler" into a "Controller".